mithril-aggregator/src/artifact_builder/cardano_database.rs

```diff
@@ -227,7 +227,7 @@ mod tests {
         }
 
         let immutable_artifact_builder = {
-            let number_of_immutable_file_loaded = beacon.immutable_file_number;
+            let number_of_immutable_file_loaded = beacon.immutable_file_number + 1;
```

Comment:
Adding a light comment explaining that this `+1` comes from the fact that we include the `00000` immutable would be appreciated.